.github/workflows/check.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: Check 🛠
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+# This reusable workflow from insightsengineering/r.pkg.template will run once a PR to main branch is created.
+jobs:
+  job-setup:
+    name: Should we run R CMD check?
+    runs-on: ubuntu-latest
+    outputs:
+      check_out: ${{ steps.check.outputs.run_r_cmd_check }}
+    steps:
+      - name: To check or not to check?
+        id: check
+        run: |
+          echo "run_r_cmd_check=false" >> $GITHUB_OUTPUT
+
+  r-cmd:
+    name: R CMD Check 🧬
+    needs: job-setup
+    # Call a workflow from a given GitHub repository, from a given branch (or tag).
+    uses: insightsengineering/r.pkg.template/.github/workflows/build-check-install.yaml@main
+    if: >
+      needs.job-setup.outputs.check_out == 'true'
```

Comment:
is this correct? isn't `check_out` equal `"run_r_cmd_check=false"`?

Reply:
The idea here was for attendees to change `false` to `true` to enable this job.